# For position, Leap Motion Controller uses '_optical' axis orientation, i.e.:
#    z forward
#    x right
#    y down

std_msgs/Header header

leap_motion_controller/Hand left_hand

leap_motion_controller/Hand right_hand

uint8 extended_fingers